lib/src/main/java/com/grio/lib/features/DeviceIdentifier.kt

```diff
@@ -0,0 +1,104 @@
+package com.grio.lib.features
+
+import android.app.Activity
+import android.content.Context.BATTERY_SERVICE
+import android.os.BatteryManager
+import android.os.Build
+import android.util.DisplayMetrics
+
+
+class DeviceIdentifier {
+
+    companion object {
+        @JvmStatic
+        fun getDeviceInformation(activity: Activity): DeviceInformation {
+
+            // Screen resolution.
+            val displayMetrics = DisplayMetrics()
+            activity.windowManager.defaultDisplay.getMetrics(displayMetrics)
+            val width = displayMetrics.widthPixels
+            val height = displayMetrics.heightPixels
+            val screenResolution = "$width x $height"
```

Comment:
string resource

Reply:
Resolving without changes as this won't be translated and the only constant String is "x".